syntax = "proto3";

package example;

service RequestService { rpc Send(Request) returns (Response); }

message Request { string msg = 1; }

message Response {
  int32 status_code = 1;
  string echo_msg = 2;
}
